public/app/features/plugins/admin/components/PluginDetailsHeader.tsx

```diff
@@ -134,5 +143,8 @@ export const getStyles = (theme: GrafanaTheme2) => {
     textUnderline: css`
       text-decoration: underline;
     `,
+    textBold: css`
```

Comment:
Maybe it's my eyes but I can't see where this style is used in this file. 🤔

Reply:
Ohh you are right, it's left here accidentally, will remove :+